hf_hub.py

```diff
@@ -43,32 +47,62 @@ def main():
     )
     args = parser.parse_args()
 
-    # Tell HuggingFace API we need to push the code for the model config and definition.
-    RADIOConfig.register_for_auto_class()
-    RADIOModel.register_for_auto_class("AutoModel")
-
     # Load the checkpoint and create the model.
     checkpoint = torch.load(args.checkpoint_path, map_location="cpu")
     model_args = checkpoint["args"]
-    radio_config = RADIOConfig(
-        vars(model_args), return_summary=True, return_spatial_features=True
-    )
-    radio_model = RADIOModel(radio_config)
+
+    if args.model == "RADIO":
+        # Tell HuggingFace API we need to push the code for the model config and definition.
+        RADIOConfig.register_for_auto_class()
+        RADIOModel.register_for_auto_class("AutoModel")
+
+        radio_model = RADIOModel(radio_config)
```

Comment:
Did you accidentally forget to include `radio_config`?

Reply:
Whoops! I fixed it in the last commit.